testproject/Assets/Scripts/Build/AndroidBuilder.cs

```diff
@@ -0,0 +1,16 @@
+using System.Linq;
+using UnityEditor;
+
+internal class AndroidBuilder
+{
+    private static void TestBuild()
+    {
+        // Build the player.\
+        var buildPlayerOptions = new BuildPlayerOptions();
+        buildPlayerOptions.scenes = EditorBuildSettings.scenes.Select(x => x.path).ToArray();
```

Comment:
@NoelStephensUnity I know you're looking to do something similar, I'm wondering if it'd be worth to consolidate both your work

Reply:
I think this is following the same pattern that we used to fix the initial issues encountered with the Multiprocess code base when we switched to using the scenes in build list as the registered scenes.  This appears to be doing the same thing by taking the scenes in build settings and applying it to the buildPlayerOptions.scenes.

Reply:
@zain-mecklai just so we don't have duplicate code ^^^